### ISSUE TITLE
AppendageNode options.controls

### ISSUE THREAD

**Issue by jonathanolson** [phetsims/john-travoltage#152]
I see:
```js
    if ( options.controls ) {
      this.setAttribute( 'aria-controls', options.controls.join( ',' ));
    }
```

but this option isn't noted in the default options, and doesn't have documentation. Can it have a default value with documentation mentioning what it does?

Code review for https://github.com/phetsims/john-travoltage/issues/128

**Comment by jonathanolson** [comment]
Also, if this is a more general pattern, should it be moved to AccessibleNode or the equivalent mixin?

**Comment by jessegreenberg** [comment]
This option has been deleted, and the line in questions has been replaced with 

```js
    // set up a relationship between the appendage and the 'status' alert so that JAWS users can quickly navigate
    // to the status element
    this.setAccessibleAttribute( 'aria-controls', AriaHerald.POLITE_STATUS_ELEMENT_ID );
```

@jonathanolson does this seem like an acceptable change?

**Comment by jessegreenberg** [comment]
Closing this issue as it is stale and setters for accessible attributes were moved to the Accessibility trait.
